app/views/admin/tags/show.html.erb

```diff
@@ -1,5 +1,7 @@
 <h1><%= @title = "Tag – #{@tag}" %></h1>
 
+<%= render partial: 'notes', locals: { notes: @notes, title: 'Notes' } %>
```

Comment:
doesn't work – not sure how to fix. might need to be `'admin/notes/show'` but then the locals don't quite work

<img width="1259" alt="Screenshot 2022-08-24 at 11 12 44" src="https://user-images.githubusercontent.com/282788/186393304-8b5086d2-c913-442c-946c-8ef51f9405e0.png">

Reply:
Ugh. I think this got excluded/missed due to my global git ignore file containing `tags` - I'll recreate.